### DISCUSSION TITLE
How to override markdown link styles?

### DISCUSSION THREAD

**Discussion by nktnet1** [fuma-nama/fumadocs#874] [Q&A] [answered]
Could you provide an example of how I can override the markdown link style?

For example, changing the text and underline (decoration ) to blue, similar to at the end of this:

![image](https://github.com/user-attachments/assets/da3c475e-f057-4a26-9548-42a3920bc1fa)

I've tried things like overriding <a> tags in global tailwind css, but that also changes the sidebar and other links. How do we override just the markdown links in `.md` and `.mdx`?

**Comment by fuma-nama** [comment, marked as answer]
We use Tailwind CSS Typography, use their utilities to customise it. Like `prose-a:`

**Reply by nktnet1** [reply]
thanks @fuma-nama - I tried something like this in global.css:

```css
.prose a {
  @apply text-blue-800 dark:text-blue-200 underline decoration-blue-800 dark:decoration-blue-200;
}
```

Which worked for the most part, but the headings also changed :(.

Couldn't figure out how to override `prose-a` specifically.

Do you know how to make it affect only links that are of the form
```md

## Don't apply style to this heading

But apply to this link here: [my-markdown-link](www.google.com)
```

**Reply by fuma-nama** [reply]
You can use :not CSS selectors for this

**Reply by nktnet1** [reply]
Thanks a lot @fuma-nama :pray: :heart: .

I ended up going with this which worked:
```css
.prose li a:not(.custom-link-style),
.prose p a:not(.custom-link-style),
.prose td a:not(.custom-link-style),
.prose blockquote a:not(.custom-link-style) {
  @apply text-blue-800 dark:text-blue-200 underline decoration-blue-800 dark:decoration-blue-200;
}
```

Not sure if there's a way to reduce repetition, but I'm generally happy with this :). Thank you!
